.travis.yml

```diff
@@ -0,0 +1,44 @@
+# This config uses industrial_ci (https://github.com/ros-industrial/industrial_ci.git).
+# For troubleshooting, see readme (https://github.com/ros-industrial/industrial_ci/blob/master/README.rst)
+
+language: generic # optional, just removes the language badge
+
+services:
+  - docker
+
+# include the following block if the C/C++ build artifacts should get cached by Travis,
+# CCACHE_DIR needs to get set as well to actually fill the cache
+cache:
+  directories:
+    - $HOME/.ccache
+
+git:
+  quiet: true # optional, silences the cloning of the target repository
+
+# limit automatic builds to certain branches (and pull requests)
+branches:
+  only:
+  - master
+
+# configure the build environment(s)
+# https://github.com/ros-industrial/industrial_ci/blob/master/doc/index.rst#variables-you-can-configure
+env:
+  global: # global settings for all jobs
+    - CCACHE_DIR=$HOME/.ccache # enables C/C++ caching in industrial_ci
+    - ROSDEP_SKIP_KEYS="log4cpp rtt ocl" # preinstalled in the docker images
+    - UPSTREAM_WORKSPACE="github:orocos/rtt_ros2_integration#master"
+  matrix: # each line is a job
+    - ROS_DISTRO="dashing" DOCKER_IMAGE="orocos/ros2:dashing-ros-base-bionic"
+    - ROS_DISTRO="eloquent" DOCKER_IMAGE="orocos/ros2:eloquent-ros-base-bionic"
+    - ROS_DISTRO="foxy" DOCKER_IMAGE="orocos/ros2:foxy-ros-base-focal"
+
+# allow failures, e.g. for unsupported distros
+# matrix:
+#   allow_failures:
+#     - env: ROS_DISTRO="lunar" ROS_REPO=ros-shadow-fixed
+
+# clone and run industrial_ci
+install:
+  - git clone --quiet --depth 1 https://github.com/ros-industrial/industrial_ci.git .industrial_ci -b master
```

Comment:
Since this is a new version of `industrial_ci.git` supporting ROS 2 and discontinuing some older features, does it mean that other files like [this](https://github.com/orocos/rtt_ros_integration/blob/c65cf80ce5ae07536fb5542b314f081e7e930d2d/.travis.yml#L41) need to be pointing to the [`legacy` branch](https://github.com/ros-industrial/industrial_ci/tree/legacy)?

Reply:
Obviously not, because it still [builds and passes successfully](https://travis-ci.org/github/orocos/rtt_ros_integration).

According to the [migration guide](https://github.com/ros-industrial/industrial_ci/blob/master/doc/migration_guide.md) the new version is mostly backwards compatible. We do not use any advanced features in [rtt_ros_integration/.travis.yml](https://github.com/orocos/rtt_ros_integration/blob/c65cf80ce5ae07536fb5542b314f081e7e930d2d/.travis.yml#L41).